test/controllers/tip_controller_test.exs

```diff
@@ -1,9 +1,10 @@
 defmodule Healthlocker.TipControllerTest do
   use Healthlocker.ConnCase
+  alias Healthlocker.Post
+  import Healthlocker.Fixtures
 
-  test "lists all entries on index", %{conn: conn} do
+  test "successfully loads tips index", %{conn: conn} do
```

Comment:
prefer a _mix_ of the old test description with the new one ... something like:
`"index lists all tips"`